draft.xml

```diff
@@ -70,27 +49,35 @@
       </address>
     </author>
 
-    <author initials="K." surname="Duleba" fullname="Krzysztof Duleba">
+    <author fullname="Krzysztof Duleba" initials="K." surname="Duleba">
       <organization>Google Switzerland GmbH</organization>
+
       <address>
         <postal>
           <street>Brandschenkestrasse 110</street>
+
           <code>8002</code>
-          <city>Zürich</city>
+
+          <city>Z?rich</city>
+
           <country>Switzerland</country>
         </postal>
 
         <email>kduleba@google.com</email>
       </address>
     </author>
 
-    <author initials="Z." surname="Szamonek" fullname="Zoltan Szamonek">
+    <author fullname="Zoltan Szamonek" initials="Z." surname="Szamonek">
       <organization>Google Switzerland GmbH</organization>
+
       <address>
         <postal>
           <street>Brandschenkestrasse 110</street>
+
           <code>8002</code>
-          <city>Zürich</city>
+
+          <city>Z?rich</city>
```

Comment:
Same

Reply:
Done in a5ec5efadd76c31a5a6b5e603b0f28289a5d0830  - see above